src/dex-ui-components/base/FormInputList/FormInputList.tsx

```diff
@@ -23,7 +26,17 @@ interface FormInputListProps<FormType extends FieldValues, FormPath extends Arra
 export function FormInputList<FormType extends FieldValues, FormPath extends ArrayPath<FormType>>(
   props: FormInputListProps<FormType, FormPath>
 ) {
-  const { fields, defaultFieldValue, formPath, register, append, remove } = props;
+  const {
+    fields,
+    defaultFieldValue,
+    formPath,
+    register,
+    append,
+    remove,
+    fieldLabel = "Member wallet address",
```

Comment:
This should be a generic component so we don't want to use defaults for these fields. These values need to be moved to the  `FormInputList` in the `MultiSigDAOGovernanceForm`.

Reply:
Done